src/Gui/decompiler-menus.xml

```diff
@@ -74,7 +74,7 @@
   <!-- Commands ========================================================== -->
 
 	<cmd id="FileOpen" cmdSet="Reko" container="FileOpen" image="0">_Open...</cmd>
-  <cmd id="FileOpenAs" cmdSet="Reko" container="FileOpenAs">Op_en As...</cmd>
+  <cmd id="FileOpenAs" cmdSet="Reko" container="FileOpenAs">Op_en As (raw)...</cmd>
```

Comment:
"Raw" is a little misleading, as some file formats are not raw, but don't contain sufficient metadata to be loaded successfully without further hints from the user. Intel HEX files come to mind: they definitely are not raw blobs of bytes, but they don't have any indication of what processor architecture or platform to use.

Reply:
Whilst in the module that did the processing of  binary file and updating the input params there was a comment with TODO, so in order to clean up the code I did!